esmvaltool/recipes/recipe_preprocessor_test.yml

```diff
@@ -15,7 +15,7 @@ preprocessors:
       # levels: {dataset: ERA-Interim, coordinate: air_pressure}
       scheme: linear
     mask_landocean:
-      fx_option: land #if land, will mask land off
+      mask_off: land #if land, will mask land off
```

Comment:
Sorry for being picky, but I think `mask_out` is the correct term. :book:

Reply:
correct, don't know what I was thinking :grin:

Reply:
replaced it, it's not bloody Zorro :grinning: